src/renderer/sw_engine/tvgSwRasterTexmap.h

```diff
@@ -58,6 +58,7 @@ static bool _arrange(const SwImage* image, const SwBBox* region, int& yStart, in
     if (yStart < regionTop) yStart = regionTop;
     if (yEnd > regionBottom) yEnd = regionBottom;
 
+    if (yEnd - yStart <= 0) return false;
```

Comment:
I think this condition is enough and we can remove the 56 line condition.
```
if (yEnd >= yStart) return false;
```

Reply:
1. 56 line is duplicated for this exceptional handling.
2. yEnd == yStart is a single line. No need to performance AA.

Reply:
please check one more time, after a minute I push changes

Reply:
l.56 removed and the current return is false for end=start, so should be fine